datasets/ttc4900/ttc4900.py

```diff
@@ -90,21 +102,18 @@ def _info(self):
             license=_LICENSE,
             # Citation for the dataset
             citation=_CITATION,
+            task_templates=[TextClassification(text_column="text", label_column="category")],
         )
 
     def _split_generators(self, dl_manager):
         """Returns SplitGenerators."""
-        path_to_manual_file = os.path.abspath(os.path.expanduser(dl_manager.manual_dir))
-        if not os.path.exists(path_to_manual_file):
-            raise FileNotFoundError(
-                "{} does not exist. Make sure you insert a manual dir via `datasets.load_dataset('ttc4900', data_dir=...)` that includes a file name {}. Manual download instructions: {})".format(
-                    path_to_manual_file, _FILENAME, self.manual_download_instructions
-                )
-            )
+
+        urls_to_download = {
+            "train": os.path.join(_DOWNLOAD_URL, _FILENAME),
```

Comment:
os.path.join uses "\" instead of "/" on windows
```suggestion
            "train": _DOWNLOAD_URL + "/" + _FILENAME,
```